isisd/isis_sr.h

```diff
@@ -1,8 +1,7 @@
 /*
- * This is an implementation of Segment Routing for IS-IS
- * as per draft draft-ietf-isis-segment-routing-extensions-25
+ * This is an implementation of Segment Routing for IS-IS as per RFC 8667
```

Comment:
nit: there are two other mentions to this draft in isisd. I'd appreciate if you could update them as well.

Reply:
Good catch. I'll update it

Reply:
Done